### ISSUE TITLE
Something is wrong with your copypasta

### ISSUE THREAD

**Issue by prempalsingh** [RadhikaG/ChatPretty#6]
Tried using ChatPretty on 2-3 conversations. Got this error on all of them. I'm sure I copied them correctly. 
Using WhatsappWeb.

**Comment by RadhikaG** [comment]
Could you send me a screenshot of the copied text? I'll correct my form validation format according to it then.

**Comment by prempalsingh** [comment]
[10/19/2015, 4:08 AM] Shreya: Awake
[10/19/2015, 4:08 AM] Shreya: ?
[10/19/2015, 9:14 AM] +91 90130 48467: Arre sorry so gaya tha
[10/19/2015, 11:07 AM] +91 90130 48467: Faced any problem?⁠⁠⁠⁠

Here's the text that I copied. The same text works completely fine when I copy it from mobile. Not working when I use Whatsapp Web.

**Comment by jailuthra** [comment]
I can reproduce this bug. WhatsappWeb uses a copypasta that is significantly different from the mobile version.

@RadhikaG I guess it's time to create a function which detects the type of copypasta and returns regexp(s) to parse that particular type. We shouldn't integrate this one in the existing regexp set, it would become too complicated. Modularizing this will also ease out adding support for non-WhatsApp chats in future. Whatsay? I'm ready to take this up :smiley:

**Comment by RadhikaG** [comment]
Yeah, time to modularize it. The current regexp set is getting unwieldy and hard to handle. Let's get down to it after the vacations, next week.
